NEWS.md

```diff
@@ -1,3 +1,11 @@
+v0.24 (2016-08-22)
+------------------
+- The `fixed` key is now recognized in limits
```

Comment:
I would expand on this a bit more, something like: "When specifying flux bounds in `media` and `limits`, the `fixed` key can now be used when lower and upper limits are the same."